st2common/bin/st2-check-license

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+if [[ $# -eq 0 ]] ; then
+echo 'No License Key Entered'
+echo 'Enter License Key'
+exit 0
+fi
+LICENSE=$@
+BWC_LICENSE_CHECK_URL="https://${LICENSE}:@packagecloud.io/install/repositories/StackStorm/enterprise/script.deb.sh"
+echo $LICENSE
+echo $BWC_LICENSE_CHECK_URL
+LICENSE_CURL_TEST=`curl --output /dev/null --silent --fail ${BWC_LICENSE_CHECK_URL}`
+if [ $? -ne 0 ]; then
```

Comment:
Instead of $?, shouldn't you be checking the value of variable LICENSE_CURL_TEST?